### ISSUE TITLE
Winning does not work

### ISSUE THREAD

**Issue by Dylan-Brisco** [hjarrell/chaotic-card#61]
Cannot determine if or which player won the game.

**Comment by supergoa** [comment]
This issue is due to the Server never requesting cards from the player. The logic in BS works for this particular issue.

**Comment by hjarrell** [comment]
If the server was not requesting the cards you wouldn't be able to play at
all.

**Comment by supergoa** [comment]
Cards are never added to cardsToPlay

**Comment by hjarrell** [comment]
@supergoa Player and BSGame are entirely network side. I see what you mean that the cards are never added to cardsToPlay but since it's BS you can always play your cards. The cards just need to be taken out of the players deck when they play them.

**Comment by supergoa** [comment]
You're right, but for a game like difungo, it would need to be coded on the network's side.

**Comment by hjarrell** [comment]
@supergoa You are right i am coding the network for difungo right now.

**Comment by supergoa** [comment]
@hjarrell Is there something you want me to do tonight?

**Comment by hjarrell** [comment]
@supergoa Work with dylan and figure out #61 this issue.

**Comment by supergoa** [comment]
@hjarrell Can you please try to get in contact with Dylan? He isn't responding to me.

**Comment by hjarrell** [comment]
@supergoa swim

**Comment by supergoa** [comment]
@hjarrell what?

**Comment by supergoa** [comment]
@hjarrell oh nevermind

**Comment by supergoa** [comment]
There is nothing to take out of the Player's hand since the network never tells BS what cards were played.

**Comment by hjarrell** [comment]
@supergoa Are you looking at GameServerClient?

**Comment by supergoa** [comment]
No, Dylan and I are looking at BS. We need to remove the played cards. But BSGame never has the information to do that.

**Comment by hjarrell** [comment]
@supergoa https://github.com/hjarrell/chaotic-card/blob/master/src/main/java/chaotic/card/networking/GameServerClient.java#L58

**Comment by supergoa** [comment]
We see that line, we don't know from which class to call it or where to use it.

**Comment by hjarrell** [comment]
@supergoa The player's cards to play are updated
